SPECS/llvm/llvm.spec

```diff
@@ -1,7 +1,11 @@
+%global maj_ver 18
+%global min_ver 1
+%global patch_ver 2
```

Comment:
Could you also modify the `Version` tag below, so we don't have two sources of truth regarding the version numbers?

Reply:
Sure, will update